index.js

```diff
@@ -35,14 +35,23 @@ function MultiMath(options) {
 }
 
 
+MultiMath.prototype.has_wasm = function () {
+  if (this.__has_wasm === null) {
```

Comment:
If this cache need? It already exists in function https://github.com/nodeca/multimath/blob/master/lib/wa_detect.js#L13